packages/components/src/components/accordion/accordion.ts

```diff
@@ -172,26 +207,9 @@ export default class SdAccordion extends SolidElement {
               'w-1 bg-accent absolute left-0 transition-all h-[calc(100%-16px)] group-hover:h-full'
             )}
           ></div>
-          <slot name="summary" part="summary" class="flex flex-auto items-center text-left">${this.summary}</slot>
-
-          <span
-            part="summary-icon"
-            class=${cx(
-              'flex flex-grow-0 flex-shrink-0 flex-auto items-center transition-all ease-in-out duration-300 text-xl',
-              this.open && 'rotate-180'
-            )}
-          >
-            <slot name="expand-icon" class=${cx(this.open && 'hidden')}>
-              <sd-icon library="system" name="chevron-down"></sd-icon>
-            </slot>
-            <slot name="collapse-icon" class=${cx(!this.open && 'hidden')}>
-              <sd-icon library="system" name="chevron-down"></sd-icon>
-            </slot>
-          </span>
+          ${this.RenderQuickfactIcon()} ${this.RenderSummary()} ${this.RenderSummaryIcons()}
```

Comment:
Ideally the accordion shouldn't know anything about the Quickfact. Maybe we can make this more neutral or something like "OptionalIcon" pr similar.